sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -109,6 +118,48 @@ constexpr cache_hint getCacheHintForIntrin() {
   }
 }
 
+const llvm::StringRef SubGroupFuncsResolution::SUB_GROUP_SHUFFLE = "__builtin_IB_simd_shuffle";
+const llvm::StringRef SubGroupFuncsResolution::SUB_GROUP_SHUFFLE_B = "__builtin_IB_simd_shuffle_b";
+const llvm::StringRef SubGroupFuncsResolution::SUB_GROUP_SHUFFLE_C = "__builtin_IB_simd_shuffle_c";
+const llvm::StringRef SubGroupFuncsResolution::SUB_GROUP_SHUFFLE_US = "__builtin_IB_simd_shuffle_us";
+const llvm::StringRef SubGroupFuncsResolution::SUB_GROUP_SHUFFLE_F = "__builtin_IB_simd_shuffle_f";
+const llvm::StringRef SubGroupFuncsResolution::SUB_GROUP_SHUFFLE_H = "__builtin_IB_simd_shuffle_h";
+const llvm::StringRef SubGroupFuncsResolution::SUB_GROUP_SHUFFLE_DF = "__builtin_IB_simd_shuffle_df";
+// b=bool, c=uchar, us=unsigned short, f=float, h=half, df=double
+//  res = intel_sub_group_shuffle(data_arg, index_arg):
+//
+//   %res = pisa.shfl.idx(%data_arg, %index, %data_arg, -1)
+
+//  res = intel_sub_group_shuffle_df(double %data_f64, i32 %index):
+//
+//   %data_i64 = bitcast double %data_f64 to <3 x i32>
```

Comment:
```suggestion
//   %data_i64 = bitcast double %data_f64 to <2 x i32>
```

Reply:
PR-OWNER: Fixed in 5786111a9f7ac69349f86aa722a0c5069e25987d

Reply:
PR-OWNER: Sorry, forgot to fix it, the actual fix is here: https://github.com/intel/llvm/commit/07f5fb9326a95b43969bf8dd691836d9ddfe53c1